results.tex

```diff
@@ -159,21 +160,22 @@ \subsection{French \gls{SFR} Deployment}
 Figure \ref{fig:pu_no_cum} shows the separated plutonium discharge per 
 month from the reprocessing plant. The plutonium outflux does not precisely 
 follow the fuel demand because \Cyclus agents have material buffers that 
-store commodity fuel for later usage. The reprocessed plutonium from legacy 
+store commodity fuel for later usage. 
 \gls{UNF} is stored for the initial loading of \glspl{SFR}.  Plutonium separated 
 from legacy \gls{UNF} meets plutonium demands sufficiently to reduce the 
 reprocessing demand for the first aggressive deployment of \glspl{SFR}.  
 The plutonium from reprocessing legacy fuel is a flat rectangle because the 
 reprocessing throughput was set to 183.2 $\frac{\gls{MTHM}}{month}$ to 
-avoid reprocessing all the legacy in one timestep. 
+avoid reprocessing all the legacy in one timestep.
  
 
 \begin{figure}[htbp!]
 	\begin{center}
 		\includegraphics[scale=0.7]{./images/french-transition/pu.png}
 	\end{center}
 	\caption{The separated plutonium discharge from the reprocessing plant 
-        in $\frac{\mbox{MTHM}}{\mbox{month}}$.}
+        in $\frac{\mbox{MTHM}}{\mbox{month}}$. The plutonium from \gls{LWR} \gls{UNF}
+        is created after the demand is gone, due to material buffers in \Cyclus.}
```

Comment:
It's not created after demand is gone, so much as stored extra Pu is transferred somewhat afterward, as it has been stockpiled... right?

Reply:
I'm not sure if I understand the question, but what is happening is that the reprocessing plant will continue to create and push plutonium if there is a request for it. The request doesn't stop until the Mixer's plutonium buffer is full. For this to happen the mixer has to stop producing fuel, and that its stockpile of produced fuel fills the buffer. In other words:
```
No longer demand for ASTRID fuel from uox plutonium
->
ASTRID fuel mixer from UOX plutonium creates ASTRID fuel until its fuel buffer is full
->
ASTRID fuel mixer stops making ASTRID fuel because buffer is full
->
Reprocessing plant keeps sending plutonium until Mixer's buffer is full
```

Reply:
Given this I think I need to clearly communicate that this plutonium production is not what is mentioned in the metrics, and that the LWR UNF reprocessed was back-calculated from the ASTRID fuel demand.